shell/client/accounts/account-settings.html

```diff
@@ -124,6 +124,42 @@ <h3 class="title-bar">Primary e-mail address for service-related notifications</
   {{> billingSettings db=db}}
   </div>
   {{/if}}
+
+  {{#if showDeletePopup}}
+    {{#modalDialogWithBackdrop onDismiss=cancelDelete class="user-confirm-delete"}}
+      {{#if deleteError}}
+        {{#focusingErrorBox}}
+          {{deleteError}}
+        {{/focusingErrorBox}}
+      {{/if}}
+
+      <h2>Delete your account</h2>
+      <p>Deleting this user account will:</p>
+      <ul>
+        <li>Permanently delete any grains created by you</li>
+        <li>Revoke any sharing links created by you</li>
+        <li>Remove you from all users' contact lists</li>
+        <li>Delete all identities owned by this account (except identities associated with another account)</li>
+      </ul>
+      <p>
+        <strong>Account deletion is permanent and irreversible.</strong>  Please be certain.
```

Comment:
This copy is kinda goofy - "permanent and irreversible", yet "will first be suspended for 7 days".  Maybe something like:

------

You are about to mark your account for deletion.  Once marked for deletion, your account will be suspended, and you will have 7 days to cancel the pending deletion by logging back into your account.

After 7 days, your account will be permanently deleted.  Deleting your account will:

* Permanently delete any grains you created
* Revoke all sharing links you created
* Remove you from all users' contact lists
* Delete all identities owned by your account (except identities also associated with another account).

<strong>Account deletion is permanent and irreversible.</strong>  Please be certain.

Type "Yes, delete my account" below to confirm.

------

A v2 of this might show precisely which identities would be deleted, and which ones would not, by using the `accountsOfIdentity` subscription.